core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/ExoPlayerDrmSessionEventListener.java

```diff
@@ -0,0 +1,43 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.drm.DefaultDrmSessionManager;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class ExoPlayerDrmSessionEventListener implements DefaultDrmSessionManager.EventListener {
```

Comment:
I think we were naming classes like this as `Listeners` ? but I am not sure if that was for the class or for the field when using it